.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: php
 
-php: 5.6
+php: 7.2
```

Comment:
The value depends on the code base and somewhat on the infrastructure where this is supposed to run in production but why `7.2` in particular - given `7.4` was released yesterday (just as an example)?

Reply:
Discussed this in person. No deeper philosophical reason.
`7.2` is the least we can do for the phpunit version specified in `composer.json` ([example problem](https://travis-ci.org/wmde/wikiba.se/builds/622690762?utm_source=github_status&utm_medium=notification)).